tsfile/src/main/java/org/apache/iotdb/tsfile/file/header/PageHeader.java

```diff
@@ -60,6 +61,10 @@ public static int calculatePageHeaderSize(TSDataType type) {
     return 3 * Integer.BYTES + 2 * Long.BYTES + Statistics.getStatsByType(type).getSerializedSize();
```

Comment:
change this with calculatePageHeaderSizeWithoutStatistics()+Statistics.getStatsByType(type).getSerializedSize();

Reply:
fixed